plugins/inventory/xen_orchestra.py

```diff
@@ -57,6 +57,14 @@
             description: Use wss when connecting to the Xen Orchestra API
             type: boolean
             default: true
+        use_vm_uuids:
+            description: Import Xen VMs to inventory using their UUID as the VM entry name (If set to false use VM name labels instead of UUID)
```

Comment:
```suggestion
            description:
              - Import Xen VMs to inventory using their UUID as the VM entry name.
              - If set to V(false) use VM name labels instead of UUID.
```